src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -353,21 +372,23 @@ function ReportActionItemMessageEdit(props) {
                             onFocus={() => {
                                 setIsFocused(true);
                                 reportScrollManager.scrollToIndex({animated: true, index: props.index}, true);
-                                ComposerActions.setShouldShowComposeInput(false);
+                                setShouldShowComposeInputKeyboardAware(false);
+
+                                // Clear active report action when another action gets focused
+                                if (!EmojiPickerAction.isActive(props.action.reportActionID)) {
+                                    EmojiPickerAction.clearActive();
+                                }
+                                if (!ReportActionContextMenu.isActiveReportAction(props.action.reportActionID)) {
+                                    ReportActionContextMenu.clearActiveReportAction();
+                                }
```

Comment:
No. We should keep active report action until another draft gets focused. This wasn't already confirmed?

Reply:
@s-alves10 This is not resolved yet. Can we do this any other way?

Reply:
`Clear active report action when another action gets focused`
We shouldn't clear active action on blur because blurred action can be active.

I think we confirmed this in the [proposal](https://github.com/Expensify/App/issues/23908#issuecomment-1657736680)

Note: I guess we can clear active in `onModalHide` of `EmojiPickerButton` for `EmojiPicker`, and in confirm/cancel callback of `showDeleteModal` for `ReportActionContextMenu`. Please let me know your thoughts

Reply:
@s77rt 

I think the new approach can cause a regression.

Reply:
I see. I don't think this is a blocker for now. Will need to test though